### DISCUSSION TITLE
Vroom formula missing dependency

### DISCUSSION THREAD

**Discussion by SimonBradley1993** [Homebrew/discussions#4676] [Writing Formulae/Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.1-47-gc7e48ce
ORIGIN: https://github.com/Homebrew/brew
HEAD: c7e48ceebbd14dc0e9c7c578ed10f2f0c9e5447a
Last commit: 28 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 75fcb1a455ceeccbfcce2c47f66f20208e041abd
Core tap last commit: 25 hours ago
Core tap branch: master
Core tap JSON: 26 Jul 14:46 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4.1-arm64
CLT: 15.0.0.0.1.1686949477
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I have installed the Vroom formula : https://formulae.brew.sh/formula/ which has 2 modes when running it. The standard mode works as expected but when using `plan` mode it requires `libglpk` to be installed.

I get this error when running
```
[Error] VROOM compiled without libglpk installed.
```

It looks like Vroom needs this package as a dependency at build time.

**Comment by osalbahr** [comment]
I recommend taking a look at [How to Open a Pull Request (and get it merged)](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request). It will guide you through the process of editing a formula. Let me know if you have any questions.
